docs/examples.rst

```diff
@@ -362,3 +362,74 @@ Calculate the token count from the token balance of an account address.
+    address='0x89205A3A3b2A69De6Dbf7f01ED13B2108B2c43e7'
+    newContract = w3.eth.contract(abi=ERC20_ABI, address=address)
+    print('Specific token address details ', newContract.functions.balanceOf(address).call())
+
+
+Query account balance
+---------------------
+
+.. code-block:: python
+    
+    import os
+    from web3.auto.infura import w3
+    import json
+    # Please play nice and don't use this key for any shenanigans, thanks!!
+    os.environ['INFURA_API_KEY'] = "b2679bb624354d1b9a2586154651b51f"
+
+    ERC20_ABI = json.loads('[{"constant":true,"inputs":[],"name":"name","outputs":[{"name":"","type":"string"}],"payable":false,"stateMutability":"view","type":"function"},{"constant":false,"inputs":[{"name":"_spender","type":"address"},{"name":"_value","type":"uint256"}],"name":"approve","outputs":[{"name":"","type":"bool"}],"payable":false,"stateMutability":"nonpayable","type":"function"},{"constant":true,"inputs":[],"name":"totalSupply","outputs":[{"name":"","type":"uint256"}],"payable":false,"stateMutability":"view","type":"function"},{"constant":false,"inputs":[{"name":"_from","type":"address"},{"name":"_to","type":"address"},{"name":"_value","type":"uint256"}],"name":"transferFrom","outputs":[{"name":"","type":"bool"}],"payable":false,"stateMutability":"nonpayable","type":"function"},{"constant":true,"inputs":[],"name":"decimals","outputs":[{"name":"","type":"uint8"}],"payable":false,"stateMutability":"view","type":"function"},{"constant":true,"inputs":[{"name":"_owner","type":"address"}],"name":"balanceOf","outputs":[{"name":"","type":"uint256"}],"payable":false,"stateMutability":"view","type":"function"},{"constant":true,"inputs":[],"name":"symbol","outputs":[{"name":"","type":"string"}],"payable":false,"stateMutability":"view","type":"function"},{"constant":false,"inputs":[{"name":"_to","type":"address"},{"name":"_value","type":"uint256"}],"name":"transfer","outputs":[{"name":"","type":"bool"}],"payable":false,"stateMutability":"nonpayable","type":"function"},{"constant":true,"inputs":[{"name":"_owner","type":"address"},{"name":"_spender","type":"address"}],"name":"allowance","outputs":[{"name":"","type":"uint256"}],"payable":false,"stateMutability":"view","type":"function"},{"anonymous":false,"inputs":[{"indexed":true,"name":"_from","type":"address"},{"indexed":true,"name":"_to","type":"address"},{"indexed":false,"name":"_value","type":"uint256"}],"name":"Transfer","type":"event"},{"anonymous":false,"inputs":[{"indexed":true,"name":"_owner","type":"address"},{"indexed":true,"name":"_spender","type":"address"},{"indexed":false,"name":"_value","type":"uint256"}],"name":"Approval","type":"event"}]')  # noqa: 501
```

Comment:
Can you elaborate more on this? An example will be much helpful here

Reply:
suppose I had an encoded JSON string which represents the ABI.

```python
abi = "<some-encoded-JSON-string-of-the-ABI>"
w3 = Web3(...)

# these two statements are equivalent
contract = w3.eth.contract(abi=abi, ...)
contract = w3.eth.contract(abi=json.loads(abi))
```

When the `w3.eth.contract` API is given a string for the `abi` parameter, it will do the JSON parsing internally.

Reply:
Do you mean I should not have the long test of ABI? Replace it with randon string?